hack/docker.nix

```diff
@@ -1,6 +1,13 @@
-{ config, lib, pkgs, controller, ... }:
+ { pkgs ? import ./nixpkgs.nix { }, ... }:
 
-pkgs.dockerTools.buildLayeredImage {
-  name = "lockbox-controller";
-  config.Entrypoint = [ "${controller}/bin/lockbox-controller" ];
-}
+ with pkgs;
+
+ let package = callPackage ./derivation.nix { };
+
+ in dockerTools.buildLayeredImage {
+   name = "lockbox-controller";
+   config = {
+     Entrypoint = [ "${package.lockbox}/bin/lockbox-controller" ];
+     Env = [ "NIX_SSL_CERT_FILE=${cacert}/etc/ssl/certs/ca-bundle.crt" ];
```

Comment:
We might not need the CA bundle in this controller, but keeping this symmetrical with `origin-ca-issuer` seems reasonable to me (for now)